NEWS.md

```diff
@@ -1,3 +1,18 @@
+# nmrec 0.2.0
+
+## New features
+
+* New convenience function `set_record_option` provides an interface
```

Comment:
I should add `()` after the function name so that pkgdown links it to the function's docs.